### PR TITLE
2nd Assignment by 202403234(Jihyun Yang)

### PR DESCRIPTION
**Pull request by angjihyun**
과제 제출합니다.

# Assignment Submission

## Student Information
- **Student ID**: 202403234
- **Name**: jihyun yang
- **Assignment**: Assignment2

## Changed Files

```
[FILE_TREE_PLACEHOLDER]
```

### Problem 1
- [ ] Implemented required functionality
- [ ] All tests pass locally

### Problem 2
- [ ] Implemented required functionality
- [ ] All tests pass locally

### Problem 3
- [ ] Implemented required functionality
- [ ] All tests pass locally

### Problem 4 (Optional)
- [ ] Implemented required functionality
- [ ] All tests pass locally

### Problem 5 (Optional)
- [ ] Implemented required functionality
- [ ] All tests pass locally

### Problem 6 (Optional)
- [ ] Implemented required functionality
- [ ] All tests pass locally

## Testing
- [ ] I have run `python grader.py` locally
- [ ] All implemented problems pass their tests
- [ ] I have tested edge cases mentioned in the problem statements

## Additional Notes

---
**Note**: This PR will be automatically validated for:
- Correct directory structure
- Valid Python syntax
- PR title format: `Nth Assignment by {student_id} (Full Name)`
- File organization compliance

### CODE REVIEW

**Comment by hist0613** [comment]
`assignments/assignment2/problem2/README.md` 를 변경했습니다.

```bash
# 기존
cd problem2/textops & python main.py

# 변경
cd problem2 & python -m textops.main
```

요렇게 실행하시면 됩니다 :)

**Comment by angjihyun** [comment]
교수님 problem4가 실행이 안되는 이유를 못찾겠습니다..

**Comment by hist0613** [comment]
ㅎㅎ... 요렇게 업데이트 했습니다 죄송..

```
# Problem 4 테스트
cd problem4 && python -m dsops.main

# Problem 5 테스트
cd problem5 && python -m cachekit.main
```

**Comment by hist0613** [comment]
`members/`를 지우시면 안 됩니다~

File: submissions/202403234/assignment2/problem2/textops/tokenize/word.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    """
+    Split on single spaces; empty/whitespace-only -> [].
+    Assume `s` is already normalized by clean_text.
+    """
+    # TODO: 구현하세요
+    # 힌트:
+    # 1) 빈 문자열이나 공백만 있는 경우 체크: if not s or s.strip() == ""
+    # 2) 단일 공백으로 분할: s.split(" ")
+    # 3) 빈 리스트 반환 조건 잊지 말기
+    raise NotImplementedError
```
Comment (hist0613) [inline submissions/202403234/assignment2/problem2/textops/tokenize/word.py:11]: 구현 미완료 상태입니다
